cmd/server/README.md

```diff
@@ -12,6 +12,7 @@ configuring the following environment variables:
 | THREESCALE_LISTEN_ADDR           | Sets the listen address for the gRPC server                                                        | 0       |
 | THREESCALE_LOG_LEVEL             | Sets the minimum log output level. Accepted values are one of `debug`,`info`,`warn`,`error`,`none` | info    |
 | THREESCALE_LOG_JSON              | Controls whether the log is formatted as JSON                                                      | true    |
+| THREESCALE_LOG_GRPC              | Controls whether the log omits gRPC info                                                           | false   |
```

Comment:
When I switch this to `true` it will print gRPC info, right? If so then maybe reword to "whether the log includes gRPC info"?

Reply:
Yes correct, will reword